docs/basic-features/data-fetching/incremental-static-regeneration.md

```diff
@@ -80,7 +80,7 @@ When a request is made to a page that was pre-rendered at build time, it will in
 
 - Any requests to the page after the initial request and before 10 seconds are also cached and instantaneous.
 - After the 10-second window, the next request will still show the cached (stale) page
-- Next.js triggers a regeneration of the page in the background.
+- Next.js triggers a regeneration of the page in the background. (May not work depending on your OS for background handling - works when deploying on Vercel)
```

Comment:
This should work regardless of your OS, can you provide additional details where you are seeing this not work?

Reply:
Yes, using this in docker and the image is `mhart/alpine-node:16.2.0`. The first request can call getStaticProps when getStaticPaths is set to `fallback`. With `revalidate` the headers are pass properly for SWR but, the server does not handle the background step after it already has the cached content to invalidate.

Reply:
Can you create a minimal reproduction on a new issue so we can investigate this? This should work fine inside of docker or any other OS so I don't think we want to document this and instead want to ensure it's working regardless of the env.

Reply:
Yes, I can get to this during the weekend np.

Reply:
Great, I'm gonna close this in favor of an issue instead.